modules/util/css-builder/src/main/java/com/liferay/css/builder/ant/BuildCSSTask.java

```diff
@@ -59,7 +59,8 @@ public void setDirNames(String dirNames) {
 	}
 
 	public void setDocrootDirName(String docrootDirName) {
-		_cssBuilderArgs.setDocrootDirName(docrootDirName);
+		_cssBuilderArgs.setDocrootDir(
+			new File(getProject().getBaseDir(), docrootDirName));
```

Comment:
Avoid chaining. Also, Ant tasks supports `File` attributes, as explained [here](https://ant.apache.org/manual/develop.html). Unless we want to maintain backwards compatibility (but we're having a major increase anyway, and I doubt anyway is using this task), we can directly expose `docrootDir`. Do you agree?